launchpod/src/main/webapp/angular-launchpod/src/app/app.component.css

```diff
@@ -0,0 +1,60 @@
+.center-page {
+    left: 50%;
+    -ms-transform: translate(-50%, -50%);
+    position: absolute;
+    top: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.icons{
+    padding-right: 10px;
+}
+
+.logo{
+    height:50px;
```

Comment:
For consistency sake, can we always add a space after the colon? :)

Reply:
done.